src/Compilers/Core/Portable/CodeGen/MetadataSerializedType.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeGen
+{
+    /// <summary>
+    /// When erasing an extension type E with underlying type U,
```

Comment:
Probably unrelated question: did we want to use just one of the term "underlying type" / "extended type" or both interchangeably?

Reply:
They are interchangeable, but you're correct to call this out. I've been trying to align on "extended type" but old habits die hard ;-)